src/IssueViz.UnitTests/Models/AnalysisIssueLocationVisualizationTests.cs

```diff
@@ -51,11 +51,65 @@ public void Ctor_InitialFilePathIsTakenFromIssueLocation()
         [TestMethod]
         public void Ctor_InitialSpanIsNull()
         {
-            var testSubject = CreateTestSubject();
+            var testSubject = CreateTestSubject(filePath:"test");
 
             testSubject.Span.Should().BeNull();
         }
 
+        [TestMethod]
```

Comment:
Very blatant copy-paste of the tests in AnalysisIssueVisualizationTests...

Reply:
It's almost as if the class structure is wrong, and `IAnalysisIssueVisualization` should _have_ an `IAnalysisIssueLocationVisualiztion` rather than _be_ an `IAnalysisIssueLocationVisualization`...

Reply:
https://github.com/SonarSource/sonarlint-visualstudio/issues/2132